android-kit-base/src/test/kotlin/com/mparticle/kits/KitManagerImplTest.kt

```diff
@@ -1451,6 +1451,10 @@ class KitManagerImplTest {
         ) {
             println("Executed with $attributes")
         }
+
+        override fun purchaseFinalized(placementId: String, catalogItemId: String, status: Boolean) {
+            TODO("Not yet implemented")
```

Comment:
This is a nonblocking comment, but rather for my knowledge.

Can you explain this construct to me of having TODO("Not yet implemented") in various methods in this file?  I see that is't not just this one, so you are being consistent, but I also see `println('Executed with $attributes)` [right above here](https://github.com/mParticle/mparticle-android-sdk/pull/583/files#diff-f0dbae0f2ddfdc859d63ae13e8e367d47f460ba2e533623da17855c83d3bf067R1443-R1453).

I assume it's used to not break the test?  Why do `println` in one place, and `TODO` in another/